.golangci.yml

```diff
@@ -43,6 +43,7 @@ linters-settings:
   misspell:
     ignore-words:
       - mosquitto
+      - creater
```

Comment:
Do we really want to add this to `kubeedge`? originally this structure member name is defined in `vendor/k8s.io/apiserver`, not `kubeedge`?

Reply:
Lint thinks `creater` is a misspelling, which should be `creator`. However, even variables in k8s are named with `creater`, so we need lint to ignore this mistake.